convertor/lib/models/price_reduction.dart

```diff
@@ -0,0 +1,22 @@
+class PriceReduction{
```

Comment:
```suggestion
class PriceReduction {
```
Un peu d'espace pour plus de propreté